assets/templates/homepage/census-tile.tmpl

```diff
@@ -1,4 +1,26 @@
-<article class="tile tile__content promo__background--plum-gradient col col--lg-29 margin-top-lg--2 margin-top-md--2 margin-left-lg--1 {{if eq .Language "en" }}height-lg--31 {{ else }}height-lg--41 {{end}}hide--sm">
+{{ if .FeatureFlags.EnableCensusTile }}
+<article class="tile tile__content promo__background--plum-gradient col col--lg-29 margin-top-lg--2 margin-top-md--2 margin-left-lg--1 height-lg--34 hide--sm">
+    <header class="margin-top--1">
+        <h2 class="margin-top--0 margin-right--0 margin-bottom--0 margin-left--0">
+                {{if eq .Language "en" }}
+                    <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-white-landscape.svg" alt="Census 2021" class="header__svg-logo"
+                    xmlns="http://www.w3.org/2000/svg" focusable="false" width="209" height="40" viewBox="0 0 228 44">
```

Comment:
`viewBox` is not an "allowed" attribute on `<img>` but not sure how you would render your svg as it's coming from the CDN